src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const correctTarget = (target: string) => {
```

Comment:
better naming, pls 🙂 
```suggestion
  const doesMoviePropertyMatchQuery = (movieProperty: string) => {
```